openstack/datadog_checks/openstack/data/conf.yaml.example

```diff
@@ -1,90 +1,130 @@
+      ## Regex expressions for the server IDs are supported.
+      #
       # exclude_server_ids:
       #    - server_1
       #    - other_.*
 
-      # IDs of networks to exclude from monitoring (by default the agent will collect metrics from networks returned by the neutron:get_networks operation)
-      # Regex expressions for the network IDs to exclude are supported.
+
+      ## @param exclude_network_ids - optional - list - default:[]
+      ## IDs of networks to exclude from monitoring (by default the agent will collect metrics from networks returned by the neutron:get_networks operation)
+      ## Regex expressions for the network IDs to exclude are supported.
+      #
       # exclude_network_ids:
       #    - network_1
       #    - other_.*
 
-      # Nova API version to use - this check supports v2 and v2.1 (default)
+      ## @param nova_api_version - optional - string - default:'v2.1'
+      ## Nova API version to use - this check supports v2 and v2.1 (default)
+      #
       # nova_api_version: 'v2.1'
 
-      # Whether to enable SSL certificate verification for HTTP requests. Defaults to true, you may
-      # need to set to false when using self-signed certs
+      ## @param ssl_verify - optional - bool - default:true
+      ## Whether to enable SSL certificate verification for HTTP requests. Defaults to true, you may
+      ## need to set to false when using self-signed certs
+      #
       # ssl_verify: true
 
-     # Whether the dd-agent proxy should also be used for openstack API requests (if set)
+     ## @param use_agent_proxy - optional - bool - default:true
+     ## Whether the dd-agent proxy should also be used for openstack API requests (if set)
+     #
      # use_agent_proxy: true
 
 instances:
-    - name: instance_1 # A required unique identifier for this instance
+    ## @param name - required - string
+    ## Unique identifier for this instance
+    #
+    - name: instance_1
+
 
-      # User credentials
-      # Password authentication is the only auth method supported
-      # User expects username, password, and user domain id
-      # `user` should resolve to a structure like {'password': 'my_password', 'name': 'my_name', 'domain': {'id': 'my_domain_id'}}
-      # The check will use the Unscoped token method to collect information about all available projects to the user
+      ## @param user - required - user_object
+      ## Password authentication is the only auth method supported
+      ## User expects username, password, and user domain id
+      ## `user` should resolve to a structure like {'password': 'my_password', 'name': 'my_name', 'domain': {'id': 'my_domain_id'}}
+      ## The check will use the Unscoped token method to collect information about all available projects to the user
+      #
       user:
-          password: my_password
-          name: datadog
+          password: <my_password>
```

Comment:
I think placeholders need to be uppercase now

Reply:
My thoughts here are to get this PR in a good shape code wise, and we can iteratively fix all the little stuff before releasing the next version of the check.